testing/endtoend/BUILD.bazel

```diff
@@ -117,6 +119,39 @@ go_test(
     deps = common_deps,
 )
 
+# gazelle:ignore
+go_test(
```

Comment:
I needed to create a new test target as we ideally want to run this in postsubmit rather than slow down the pre-submit pipeline